src/command.c

```diff
@@ -672,6 +674,27 @@ int cmd_quit(struct command *cmd)
     return CORVUS_OK;
 }
 
+int cmd_select(struct command *cmd, struct redis_data *data)
+{
+    ASSERT_TYPE(data, REP_ARRAY);
+    ASSERT_ELEMENTS(data->elements == 2, data);
+
+    struct redis_data *db_data = &data->element[1];
+    ASSERT_TYPE(db_data, REP_STRING);
+
+    struct pos_array *db_str = &db_data->pos;
+    // TODO: Implement pos_to_long then use it here, someday.
+    if (db_str->str_len == 1 && db_str->items->str[0] == '0') {
```

Comment:
The comparison is not very suitable here. If the command `select 000` is issued, `OK` should be returned.